### PR TITLE
Marking interfaces as [SecureContext]

### PR DESCRIPTION
**Pull request by cwilso**
Fixes issue #183.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -159,7 +159,7 @@ <h2>Feature Policy Integration</h2>
           <h2>Extensions to the <dfn>Navigator</dfn> interface</h2>
 
           <pre class="idl">
-            partial interface Navigator {
+            [SecureContext] partial interface Navigator {
```
Comment (marcoscaceres) [inline index.html]: Oh, this is not allowed in WebIDL. You need to put [SecureContext] directly on the requestMIDIAccess() method instead.

**Review by marcoscaceres — CHANGES_REQUESTED** [review]
Just one small thing.

File: index.html [hunk 2]
```diff
@@ -832,7 +832,7 @@ <h2><dfn>MIDIPortConnectionState</dfn> Enum</h2>
       <h2 id="MIDIMessageEvent"><dfn>MIDIMessageEvent</dfn> Interface</h2>
       <p>An event object implementing this interface is passed to a MIDIInput's onmidimessage handler when MIDI messages are received.  Note that the DOM <a>Event</a> <code>timeStamp</code> attribute is defined as a <a>DOMHighResTimeStamp</a>, and represents the high-resolution time of when the event was received or is to be sent.</p>
       <pre class="idl">
-        [Constructor(DOMString type, optional MIDIMessageEventInit eventInitDict)]
+        [SecureContext] [Constructor(DOMString type, optional MIDIMessageEventInit eventInitDict)]
         interface MIDIMessageEvent: Event {
```
Comment (marcoscaceres) [inline index.html:836]: Nit: missing “,”

File: index.html [hunk 3]
```diff
@@ -887,7 +887,7 @@ <h2 id="MIDIConnectionEvent"><dfn>MIDIConnectionEvent</dfn> Interface</h2>
         for new devices infrequently.  As such, it is suggested that heavy 
         reliance on connection events not be used.</p>
       <pre class="idl">
-        [Constructor(DOMString type, optional MIDIConnectionEventInit eventInitDict)]
+        [SecureContext] [Constructor(DOMString type, optional MIDIConnectionEventInit eventInitDict)]
```
Comment (marcoscaceres) [inline index.html]: As above.

**Review by marcoscaceres — CHANGES_REQUESTED** [review]
Couple of other nits.

File: index.html (hunk 2 above)
Comment (cwilso) [reply index.html:836]: done.

**Pull request merged**